src/Checkbox.js

```diff
@@ -0,0 +1,186 @@
+import React from "react";
+import { View, TouchableOpacity, StyleSheet, Image } from "react-native";
+import PropTypes from "prop-types";
+// galio dependency
+import { Icon, Text } from ".";
+import GalioTheme, { withGalio } from "./theme";
+
+class Checkbox extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      checked: props.initialValue
+    };
+
+    this.renderChecked = this.renderChecked.bind(this);
+    this.spaceAround = this.spaceAround.bind(this);
+  }
+
+  // adding the necessary margins depending on the flexDirection
+  spaceAround(direction) {
+    switch (direction) {
+      case "row":
+        return { marginLeft: 10 };
+      case "row-reverse":
+        return { marginRight: 10 };
+      case "column":
+        return { marginTop: 10 };
+      case "column-reverse":
+        return { marginBottom: 10 };
+      default:
```

Comment:
row & default return the same value

```
case "row":
default:
    return { marginLeft: 10 };
```

Reply:
Do you recommend removing the 'row' case?

Reply:
Maybe row is the default direction